draft-ietf-idr-linklocal-capability.xml

```diff
@@ -105,7 +125,7 @@ in BGP.</t>
 <section numbered="true" toc="default">
       <name>Link-Local Next Hop Capability</name>
       <t>The Link-Local Next Hop capability is a new BGP capability. A
-BGP speaker that supports capabilities advertisement <xref target="RFC5492" format="default"/>
+BGP speaker that supports capabilities advertisement <xref target="RFC5492"/>
```

Comment:
It's weird the cleanup here is being done vs. a PR on a specific other issue, but whatever. :-)

Reply:
It (cleanup) has its own separate commit, so the history is not b0rked.